### ISSUE TITLE
Provide raw log view for a pod

### ISSUE THREAD

**Issue by jstrachan** [hawtio/hawtio-kubernetes#76]
maybe kibana is overkill for looking at the log of just 1 live pod?

**Comment by jimmidyson** [comment]
Right now this is trickier than it sounds... Logs endpoint doesn't support websockets: only spdy or chunked. Not sure of if xhr supports chunked responses? There is an open issue in kubernetes fit this IIRC, as there is for exec which has the same problem.

**Comment by jstrachan** [comment]
Agreed. Showing a static view would be OK (with an icky Refresh button) until websockets are available

**Comment by gashcrumb** [comment]
honestly a static view is probably easier to deal with from a user's perspective, i.e. it's easy to copy/paste lines from it to create a gist without having to worry about the view scrolling away on you :-)
